cmake/projects/NLopt/hunter.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    NLopt
+    VERSION
+    2.5.0-p0
+    URL
+    "https://github.com/hunter-packages/nlopt/archive/v2.5.0-p0.tar.gz"
+    SHA1
+    30838bcdc191b6dae8af1d0e8f704fe51a0ad3ef
+)
+
+hunter_cmake_args(NLopt CMAKE_ARGS
+		NLOPT_CXX=OFF
+		BUILD_SHARED_LIBS=OFF
```

Comment:
I mean it should not be touched at all. Hunter will add `BUILD_SHARED_LIBS=OFF` by default and user is free to modify it by `HUNTER_BUILD_SHARED_LIBS` or `hunter_config(... CMAKE_ARGS BUILD_SHARED_LIBS=OFF)`.

Reply:
I don't know it

Reply:
> I don't know it

You don't know what?

At this point the `BUILD_SHARED_LIBS=OFF` is at least redundant, Hunter will add `BUILD_SHARED_LIBS=OFF` by default to the package build.